apisix/core/ctx.lua

```diff
@@ -164,16 +163,15 @@ do
                     end
                 end
 
-            elseif core_str.has_prefix(key, "arg_") then
-                local args = t.args
-                local arg_key = sub_str(key, 5)
-                local m, err = ngx_re_match(args, "(^|&)" .. arg_key .. "=([^&]*)(&|$)", "jo")
-                if err then
-                    log.warn("failed to fetch arg value by key: ",
-                             key, " error: ", err)
-                end
-                if m then
-                    val = m[2]
+            elseif core_str.has_prefix(key, "uri_args_") then
```

Comment:
Err... Let's use the old name "arg_".

Reply:
Sorry for my mistake, I didn't make it clear

Reply:
It's ok